en/telemetry/microhard_serial.md

```diff
@@ -0,0 +1,53 @@
+# Microhard Serial Radio
+
+[Microhard Pico Serial](http://microhardcorp.com/P900.php) radios enable MAVLink communication between a radio on a vehicle and a GCS.
+Microhard Pico Serial radios are 1 Watt output radios that support point to point, point to multi-point, and mesh modes.
+A single ground station radio can be used to communicate with multiple vehicles using point to multi-point or mesh.
+Vehicles must have different IDs.
```

Comment:
MAVLink IDs? Or some other ID

People are interested in range these can be used at. Any ball park figures for a single GCS /vehicle system running on 1mW and 1W?

If I change nothing but the baud rate, will this set up a one to one network? 

Does the mesh network stuff "work"? I assume this just means that if you set up a mesh and you had multiple vehicles and GCS they would all see all traffic?

Do you want to talk about other benefits - e.g. encryption. How do you set that up?

Reply:
Yes Mavlink IDs. 
Ballpark for point to point at 1W with the default settings is about 5 miles with no other optimizations. 
Yes the mesh network works on these. Same caveat applies that all vehicles would need a different mavlink ID.
Encryption is an optional feature that is extra at order time. If you have a unit with encryption, the setting appears in PicoConfig.